### PR TITLE
Document port mapping caveat

### PR DESCRIPTION
**Pull request by fkorotkov**
Related to #810 and #521

### CODE REVIEW

File: docs/guide/writing-tasks.md [hunk 1]
```diff
@@ -876,6 +876,10 @@ Additional container can be very handy in many scenarios. Please check [Cirrus C
 ??? tip "Port Mapping"
     It's also possible to map ports of additional containers by using `<HOST_PORT>:<CONTAINER_PORT>` format for the `port` field.
     For example, `port: 80:8080` will map port `8080` of the container to be available on local port `80` within a task.
+  
+    **Note:** don't use port mapping if it's not absolutly nessesary. For example, when you have several additional containers
```
Comment (edigaryev) [inline docs/guide/writing-tasks.md:880]:
```suggestion
    **Note:** don't use port mapping unless absolutely necessary. For example, when you have several additional containers
```

File: docs/guide/writing-tasks.md [hunk 2]
```diff
@@ -876,6 +876,10 @@ Additional container can be very handy in many scenarios. Please check [Cirrus C
...
+    which expose the same port and you don't have control over changing the default port number. Port mapping limits
```
Comment (edigaryev) [inline docs/guide/writing-tasks.md:881]:
```suggestion
    which start the service on the same port and there's no easy way to change that. Port mapping limits 
```

File: docs/guide/writing-tasks.md [hunk 3]
```diff
@@ -876,6 +876,10 @@ Additional container can be very handy in many scenarios. Please check [Cirrus C
 ??? tip "Port Mapping"
     It's also possible to map ports of additional containers by using `<HOST_PORT>:<CONTAINER_PORT>` format for the `port` field.
     For example, `port: 80:8080` will map port `8080` of the container to be available on local port `80` within a task.
+  
+    **Note:** don't use port mapping unless absolutely necessary. For example, when you have several additional containers
+    which start the service on the same port and there's no easy way to change that. Port mapping limits
```
Comment (RDIL) [inline docs/guide/writing-tasks.md:881]: Grammar - this sentence ends too abruptly

Comment (fkorotkov) [reply docs/guide/writing-tasks.md:881]: Which one? "Port mapping limits"? There is continuation on the next line. 🤔

Comment (RDIL) [reply docs/guide/writing-tasks.md:881]: `that. Port mapping limits` You are giving an example here, and then end the sentence.  You don't explain what this example is for until the next sentence, which makes very little sense,

File: docs/guide/writing-tasks.md [hunk 4]
```diff
@@ -876,6 +876,10 @@ Additional container can be very handy in many scenarios. Please check [Cirrus C
 ??? tip "Port Mapping"
     It's also possible to map ports of additional containers by using `<HOST_PORT>:<CONTAINER_PORT>` format for the `port` field.
     For example, `port: 80:8080` will map port `8080` of the container to be available on local port `80` within a task.
+  
+    **Note:** don't use port mapping unless absolutely necessary. For example, when you have several additional containers
```
Comment (edigaryev) [inline docs/guide/writing-tasks.md:880]:
```suggestion
    **Note:** don't use port mapping unless absolutely necessary. A perfect use case is when you have several additional containers
```

Fixes abruptly ending sentence.

File: docs/guide/writing-tasks.md (hunk 3 above)
Comment (edigaryev) [reply docs/guide/writing-tasks.md:881]: See https://github.com/cirruslabs/cirrus-ci-docs/pull/811/files#r595227933 for a fix.

File: docs/guide/writing-tasks.md (hunk 4 above)
Comment (RDIL) [reply docs/guide/writing-tasks.md:880]: That is much better.

**Pull request merged**